packages/design-system/src/connect/components/document-toolbar/utils/use-document-undo-redo.ts

```diff
@@ -0,0 +1,27 @@
+import { useDocumentById } from "@powerhousedao/reactor-browser";
+import { redo, undo } from "document-model/core";
+
+export function useDocumentUndoRedo(documentId?: string) {
+  const [document, dispatch] = useDocumentById(documentId);
+
+  const globalRevisionNumber = document?.header.revision.global ?? 0;
+  const localRevisionNumber = document?.header.revision.local ?? 0;
+
+  // TODO: check why undo and redo actions are not working
```

Comment:
I've also noticed this, created a ticket: https://github.com/powerhouse-inc/powerhouse/issues/2027